cmd/operator-sdk/run/cmd.go

```diff
@@ -149,38 +151,48 @@ https://sdk.operatorframework.io/docs/olm-integration/cli-overview
 	cmd.Flags().BoolVar(&c.olm, "olm", false,
 		"The operator to be run will be managed by OLM in a cluster. "+
 			"Cannot be set with another run-type flag")
-	err := cmd.Flags().MarkDeprecated("olm", "use 'run packagemanifests' instead")
+	err = cmd.Flags().MarkDeprecated("olm", "use 'run packagemanifests' instead")
```

Comment:
`deprecated olm in favor of a new subcommand` shows not part of the scope `deprecate '--local' flag in favor of 'local' subcommand. Could not it be done in another PR? If we need to revert the specific change for olm, for example,  would not be an issue?

Reply:
This wasn't added, I only changed `:=` to `=`.